### PR TITLE
📝 feat: add pull request template

### LINKED ISSUE #5: Add a PR template

### PR DESCRIPTION
**Pull request by fuadop**
resolves #5

### CODE REVIEW

**Comment by fuadop** [comment]
Hey @JoshuaKGoldberg , I copied the exact same one at typescript-eslint you recomended. Also I saw the greeting was commented out.. I left it commented out. Is there any reason for that ?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!-- 👋 Hi, thanks for sending a PR to LearningTypescript! 💖.
+Please fill out all fields below and make sure each item is true and [x] checked.
+Otherwise we may not be able to review your PR. -->
+
+## PR Checklist
+
+-   [ ] Addresses an existing open issue: fixes #000
+-   [ ] That issue was marked as [accepting prs](https://github.com/LearningTypescript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
+-   [ ] Steps in [CONTRIBUTING.md](https://github.com/LearningTypescript/projects/blob/main/CONTRIBUTING.md) were taken
```
Comment (JoshuaKGoldberg) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Fun fact: you don't have to put these files in a `.github` directory. I just like it to keep them organized and out of the way.

```suggestion
-   [ ] Steps in [CONTRIBUTING.md](https://github.com/LearningTypeScript/projects/blob/main/.github/CONTRIBUTING.md) were taken
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!-- 👋 Hi, thanks for sending a PR to LearningTypescript! 💖.
+Please fill out all fields below and make sure each item is true and [x] checked.
+Otherwise we may not be able to review your PR. -->
+
+## PR Checklist
+
+-   [ ] Addresses an existing open issue: fixes #000
+-   [ ] That issue was marked as [accepting prs](https://github.com/LearningTypescript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
```
Comment (JoshuaKGoldberg) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: In practice this typo shouldn't impact anything, but I do like having proper casing 😁 

```suggestion
-   [ ] That issue was marked as [accepting prs](https://github.com/LearningTypeScript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
```

**Review by JoshuaKGoldberg — APPROVED** [review]
Yup, this looks great, thanks! 

Just a couple of small tweaks and it'll be good to go.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!-- 👋 Hi, thanks for sending a PR to LearningTypescript! 💖.
+Please fill out all fields below and make sure each item is true and [x] checked.
+Otherwise we may not be able to review your PR. -->
```
Comment (JoshuaKGoldberg) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: https://github.com/LearningTypeScript/projects/pull/13#issuecomment-1133926807:

> Hey @JoshuaKGoldberg , I copied the exact same one at typescript-eslint you recomended. Also I saw the greeting was commented out.. I left it commented out. Is there any reason for that ?

Yup, that's intentional. We don't need that greeting to show up in text on every PR. But people should see it when they fill out the PR description.

Comment (fuadop) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Oh I see. That's cool

**Comment by JoshuaKGoldberg** [comment]
Perfect, thanks!

**Comment by JoshuaKGoldberg** [comment]
https://github.com/all-contributors please add @fuadop for documentation

**Comment by JoshuaKGoldberg** [comment]
@all-contributors please add @fuadop for documentation

**Pull request merged**